app/validators/answers/number_answer_validator.py

```diff
@@ -1,7 +1,7 @@
 from app.validators.answers.answer_validator import AnswerValidator
 
-MAX_NUMBER = 9999999999
-MIN_NUMBER = -999999999
+MAX_NUMBER = 9_999_999_999_999
+MIN_NUMBER = -999_999_999_999
```

Comment:
This is only max 13 digits. Card say max 15 for positive, 14 for negative.

Reply:
My bad it was my assumption, since the card said we already support 12 digits with only 10 9's in the max_number variable